### DISCUSSION TITLE
测试新的商店链接

### DISCUSSION THREAD

**Discussion by cotaku** [cotaku/Bibi-Release#61] [Announcements]
https://www.microsoft.com/store/apps/9NJJ5RRCZSV2
一般人我不告诉ta，闷声发大财晓不晓得伐 <_<
过年回老家，有什么问题过完正月十五回来处理（逃

**Comment by Chia-tau** [comment]
請問大哥Windows 10 Mobile 1709 Edge進不去這個網站然後Microsoft Store找不到這個東西應該怎麼辦（大悲

**Reply by cotaku** [reply]
store里面搜不到的，只能通过链接直接进入，或者先在PC上下一次，然后手机上登同一个微软账户就可以在my app里直接下载了。
